bentoml/artifact/tf_savedmodel_artifact.py

```diff
@@ -115,6 +131,13 @@ def load(self, path):
         return self.pack(loaded_model)
 
 
+class TensorflowKerasSavedModelArtifact(TensorflowSavedModelArtifact):
```

Comment:
could you add docstring here similar to TensorflowSavedModelArtifact, with some example code?

Reply:
will do

Reply:
Will do!

Reply:
And could you also add this to the docs/artifacts list?  so it shows up in our docs site?